tests/specs/benchmarks/functional-spec.k

```diff
@@ -27,4 +27,8 @@ module FUNCTIONAL-SPEC
 
     claim <k> runLemma(#ceil32(DATA_LEN) +Int 292 <Int DATA_LEN +Int 292) => doneLemma(false) ... </k> requires #range(0 <= DATA_LEN < pow16)
 
+    claim <k> runLemma ( #buf ( 32 , keccak ( #range ( b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x80" [ 128 := ( #buf ( DATA_LEN , DATA ) ++ #buf ( ( DATA_LEN up/Int 32 ) *Int 32 -Int DATA_LEN , 0 ) ) [ 0 .. DATA_LEN ] ] , 128 , DATA_LEN ) ) ) ) =>
```

Comment:
This is surely an incorrect way of representing the "starting" byte array. How would I write this in k-friendly form? Perhaps there is a macro defined somewhere for it that I didn't find?

Reply:
But I think you can replace the entire `b"\x...."` with just `_:ByteArray`, and the test is still testing the lemmas well??

Reply:
Or you could replace it with `BA:ByteArray`, and add the side-condition `#sizeByteArray(BA) ==Int 32`.